release_notes_generator/record/factory/default_record_factory.py

```
@@ -46,53 +48,87 @@ class DefaultRecordFactory(RecordFactory):
         """
         Generate records for release notes.
         Parameters:
             data (MinedData): The MinedData instance containing repository, issues, pull requests, and commits.
         Returns:
-            dict[int|str, Record]: A dictionary of records where the key is the issue or pull request number.
+            dict[str, Record]: A dictionary of records keyed by 'owner/repo#number' (or commit SHA for commits).
         """
 
-        def register_pull_request(pull: PullRequest, skip_rec: bool) -> None:
-            detected_issues = extract_issue_numbers_from_body(pull)
+        def register_pull_request(pr: PullRequest, skip_rec: bool) -> None:
```

Ruff (0.13.1)

Unused function argument: `skip_rec`

(ARG001)